modules/gitops-creating-rolloutmanager-custom-resource.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/argo-rollouts-for-progressive-delivery-of-deployments.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-creating-rolloutmanager-custom-resource_{context}"]
+= Creating a RolloutManager custom resource
+
+To manage progressive delivery of deployments by using the Argo Rollouts in {gitops-title}, you must create and configure a `RolloutManager` custom resource (CR) in the namespace of your choice. By default, any new `argo-rollouts` instance has permission to manage resources only in the namespace where it is deployed, but you can use Argo Rollouts in multiple namespaces as required.
+
+.Prerequisites
+
+* {gitops-title} 1.9.0 or a newer version is installed in your cluster.
+
+.Procedure 
+
+. Log in to the {product-title} web console as a cluster administrator. 
+
+. In the *Administrator* perspective, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where you want to create and configure a `RolloutManager` custom resource (CR) from the *Project* drop-down menu.
+
+. Select *OpenShift GitOps Operator* from the installed operators.
+
+. In the *Details* tab, under the *Provided APIs* section, click *Create instance* in the *RolloutManager* pane.
+
+. On the *Create RolloutManager* page, select the *YAML view* and use the default YAML or edit it as per your requirements:
++
+.Example: RolloutManager CR
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: RolloutManager
+metadata:
+  name: argo-rollout
+  labels:
+    example: basic
+spec: {}
+----
+
+. Click *Create*.
+
+. In the *RolloutManager* tab, under the *RolloutManagers* section, verify that the *Status* field of the RolloutManager instance shows as *Phase: Available*.
+
+. In the left navigation pane, verify the creation of the namespace-scoped supporting resources:
++ 
+* Click *Workloads* -> *Deployments* and ensure the `argo-rollouts` deployment is available with the *Status* showing as `1 of 1 pods` running.
```

Comment:
My instinct for all of these is to suggest replacing "and ensure the" with "to verify that the"